trigger/patch.go

```diff
@@ -190,10 +190,16 @@ func (t *patchTriggers) makeData(sub *event.Subscription) (*commonTemplateData,
 	if api.ProjectIdentifier != nil {
 		projectName = utility.FromStringPtr(api.ProjectIdentifier)
 	}
-	collectiveStatus, err := patch.CollectiveStatus(t.patch.Id.Hex())
-	if err != nil {
-		return nil, errors.Wrap(err, "getting collective status for patch")
+
+	collectiveStatus := t.patch.Status
+	if t.patch.IsParent() {
+		var err error
+		collectiveStatus, err = t.patch.CollectiveStatus()
```

Comment:
do we still need the `patch.CollectiveStatus(t.patch.Id.Hex())` function or can it be removed now?

Reply:
we need that function for versions (only used in 1 place)